src/molecules/select/select-async.tsx

```diff
@@ -0,0 +1,38 @@
+import noop from 'lodash/noop'
+import React, { FC } from 'react'
+import ReactAsyncSelect, { AsyncProps } from 'react-select/async'
+import { theme } from '../..'
+import { cssClass, filterStyles, selectStyles } from '../../utils'
+
+interface SelectProps extends AsyncProps<unknown, boolean, any> {
+  value: any
+  onChange?: (selected) => void
+  variant?: 'default' | 'filter'
+}
+
+export const SelectAsync: FC<SelectProps> = (props) => {
+  const { value, onChange, variant, ...selectProps } = props
+  const styles = variant === 'filter' ? filterStyles(theme) : selectStyles(theme)
+
+  const handleChange = (selected) => {
+    if (typeof onChange === 'function') onChange(selected)
+  }
+
+  return (
+    <ReactAsyncSelect
+      className={cssClass('Select')}
+      value={value}
+      styles={styles}
+      onChange={handleChange}
+      isClearable
```

Comment:
can you override `isClearable` via `selectProps`?

Reply:
Yes, tried with example app and user resource

![image](https://user-images.githubusercontent.com/85543813/160806944-35123974-7b2a-428b-9173-59a86b6b8fca.png)
```
gender: {
  props: {
    isClearable: false,
  },
},
```
![image](https://user-images.githubusercontent.com/85543813/160807402-c8b7dc6f-17d0-4e1f-8c31-43707cbbfaa2.png)